oss-cloud-backend/databaseService.js

```diff
@@ -1,4 +1,19 @@
-var docClient = require("serverless-dynamodb-client").doc;
+var dynamodb = require("serverless-dynamodb-client");
+var rawClient = dynamodb.raw;
+var docClient = dynamodb.doc;
+var attr = require("dynamodb-data-types").AttributeValue;
+
+module.exports.getAllContributors = () => {
+  const params = {
+    TableName: "contributors"
+  };
+  return new Promise((resolve, reject) => {
+    rawClient.scan(params, (error, data) => {
+      if (error) reject(error);
```

Comment:
I would prefer the whole syntax for if else, it's more readable.

Reply:
@banecogic requested this syntax in [this pull request](https://github.com/levinine/oss-cloud/pull/5).

![image](https://user-images.githubusercontent.com/12577545/64117395-66cc5880-cd95-11e9-9898-986cd25aa384.png)